Content.Shared/Damage/Systems/PassiveDamageSystem.cs

```diff
@@ -0,0 +1,55 @@
+using Content.Shared.Damage.Components;
+using Content.Shared.Mobs.Systems;
+using Content.Shared.Mobs.Components;
+using Content.Shared.FixedPoint;
+using Robust.Shared.Timing;
+
+namespace Content.Shared.Damage;
+
+public sealed class PassiveDamageSystem : EntitySystem
+{
+    [Dependency] private readonly DamageableSystem _damageable = default!;
+    [Dependency] private readonly IGameTiming _timing = default!;
+    [Dependency] private readonly MobStateSystem _mobState = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<PassiveDamageComponent, MapInitEvent>(OnPendingMapInit);
+    }
+
+    private void OnPendingMapInit(EntityUid uid, PassiveDamageComponent component, MapInitEvent args)
+    {
+        component.NextDamage = _timing.CurTime + TimeSpan.FromSeconds(1f);
+    }
+
+    // Every tick, attempt to damage entities
+    public override void Update(float frameTime)
+    {
+        base.Update(frameTime);
+        var curTime = _timing.CurTime;
+
+        // Go through every entity with the component
+        var query = EntityQueryEnumerator<PassiveDamageComponent, DamageableComponent, MobStateComponent>();
+        while (query.MoveNext(out var uid, out var comp, out var damage, out var mobState))
+        {   
+            // Make sure they're up for a damage tick
+            if (comp.NextDamage > curTime)
+                continue;
+
+            if (comp.DamageCap != null && damage.TotalDamage > comp.DamageCap)
```

Comment:
Isn't this check slightly buggy? This will allow damage to exceed the cap. This doesn't seem like such a big deal, but if it's an easy fix you should consider looking at that.